# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-ac-build gcc-g++-bootstrap-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.64)
./configure --disable-libatomic --disable-libcilkrts --disable-libgomp --disable-libmpx --disable-libquadmath --disable-libsanitizer --disable-libssp --disable-libvtv --disable-lto --disable-multilib --disable-plugin --enable-languages=c++ --with-native-system-header-dir=/include --prefix=

# build
make

# install
install -d $DESTDIR/bin
install host-`./config.guess`/gcc/cpp $DESTDIR/bin
install host-`./config.guess`/gcc/xg++ $DESTDIR/bin/g++
ln -sf g++ $DESTDIR/bin/c++
install host-`./config.guess`/gcc/xgcc $DESTDIR/bin/gcc
ln -sf gcc $DESTDIR/bin/cc
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/algorithm $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cctype $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/clocale $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstdio $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstdlib $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cstring $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/c_global/cwchar $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/iosfwd $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/list $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/map $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/set $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/string $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/type_traits $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/include/std/utility $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/backward
install -m 644 libstdc++-v3/include/backward/binders.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/backward
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/algorithmfwd.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/allocator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/basic_string.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/basic_string.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/char_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/concept_check.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/cpp_type_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/functexcept.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/list.tcc $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/localefwd.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/move.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/ostream_insert.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/postypes.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/range_access.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_algo.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_algobase.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_construct.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_function.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_heap.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator_base_funcs.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_iterator_base_types.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_list.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_map.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_multimap.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_multiset.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_pair.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_relops.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_set.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_tempbuf.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stl_tree.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/include/bits/stringfwd.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/debug
install -m 644 libstdc++-v3/include/debug/debug.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/debug
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/alloc_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/atomicity.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/new_allocator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/numeric_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/include/ext/type_traits.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/ext
install -m 644 libstdc++-v3/libsupc++/atomic_lockfree_defines.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/cxxabi_forced.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/exception $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -m 644 libstdc++-v3/libsupc++/exception_defines.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/exception_ptr.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/nested_exception.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/bits
install -m 644 libstdc++-v3/libsupc++/new $DESTDIR/include/c++/`cat gcc/BASE-VER`
install -d $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/atomic_word.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++allocator.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++config.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/c++locale.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/cpu_defines.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/gthr-default.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/gthr.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -m 644 `./config.guess`/libstdc++-v3/include/`./config.guess`/bits/os_defines.h $DESTDIR/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits
install -d $DESTDIR/lib
ln -sf ../bin/cpp $DESTDIR/lib/cpp
install host-`./config.guess`/gcc/libgcc_s.so.1 $DESTDIR/lib
ln -sf libgcc_s.so.1 $DESTDIR/lib/libgcc_s.so
install `./config.guess`/libstdc++-v3/src/.libs/libstdc++.so.6.* $DESTDIR/lib/libstdc++.so.6
ln -sf libstdc++.so.6 $DESTDIR/lib/libstdc++.so
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtbegin.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtbeginS.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtend.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtendS.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/crtfastmath.o $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -m 644 host-`./config.guess`/gcc/libgcc.a $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/cpuid.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/emmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/float.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/ia32intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/immintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/mm_malloc.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/mmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/smmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdarg.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdbool.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stddef.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/stdnoreturn.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/tmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/unwind.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/x86intrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -m 644 host-`./config.guess`/gcc/include/xmmintrin.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include
install -d $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed
install -m 644 host-`./config.guess`/gcc/include-fixed/limits.h $DESTDIR/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed
install -d $DESTDIR/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`
install host-`./config.guess`/gcc/cc1 $DESTDIR/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`
install host-`./config.guess`/gcc/cc1plus $DESTDIR/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/gcc-g++-build.pc << EOF
Name: gcc
Version: 6.5.0
Description: GNU Compiler Collection
Requires: $REQUIRES

devel=\\
/bin/c++ \\
/bin/cc \\
/bin/cpp \\
/bin/g++ \\
/bin/gcc \\
/include/c++/`cat gcc/BASE-VER`/algorithm \\
/include/c++/`cat gcc/BASE-VER`/cctype \\
/include/c++/`cat gcc/BASE-VER`/clocale \\
/include/c++/`cat gcc/BASE-VER`/cstdio \\
/include/c++/`cat gcc/BASE-VER`/cstdlib \\
/include/c++/`cat gcc/BASE-VER`/cstring \\
/include/c++/`cat gcc/BASE-VER`/cwchar \\
/include/c++/`cat gcc/BASE-VER`/exception \\
/include/c++/`cat gcc/BASE-VER`/iosfwd \\
/include/c++/`cat gcc/BASE-VER`/list \\
/include/c++/`cat gcc/BASE-VER`/map \\
/include/c++/`cat gcc/BASE-VER`/new \\
/include/c++/`cat gcc/BASE-VER`/set \\
/include/c++/`cat gcc/BASE-VER`/string \\
/include/c++/`cat gcc/BASE-VER`/type_traits \\
/include/c++/`cat gcc/BASE-VER`/utility \\
/include/c++/`cat gcc/BASE-VER`/backward/binders.h \\
/include/c++/`cat gcc/BASE-VER`/bits/algorithmfwd.h \\
/include/c++/`cat gcc/BASE-VER`/bits/allocator.h \\
/include/c++/`cat gcc/BASE-VER`/bits/atomic_lockfree_defines.h \\
/include/c++/`cat gcc/BASE-VER`/bits/basic_string.h \\
/include/c++/`cat gcc/BASE-VER`/bits/basic_string.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/char_traits.h \\
/include/c++/`cat gcc/BASE-VER`/bits/concept_check.h \\
/include/c++/`cat gcc/BASE-VER`/bits/cpp_type_traits.h \\
/include/c++/`cat gcc/BASE-VER`/bits/cxxabi_forced.h \\
/include/c++/`cat gcc/BASE-VER`/bits/exception_defines.h \\
/include/c++/`cat gcc/BASE-VER`/bits/exception_ptr.h \\
/include/c++/`cat gcc/BASE-VER`/bits/functexcept.h \\
/include/c++/`cat gcc/BASE-VER`/bits/list.tcc \\
/include/c++/`cat gcc/BASE-VER`/bits/localefwd.h \\
/include/c++/`cat gcc/BASE-VER`/bits/move.h \\
/include/c++/`cat gcc/BASE-VER`/bits/nested_exception.h \\
/include/c++/`cat gcc/BASE-VER`/bits/ostream_insert.h \\
/include/c++/`cat gcc/BASE-VER`/bits/postypes.h \\
/include/c++/`cat gcc/BASE-VER`/bits/range_access.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_algo.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_algobase.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_construct.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_function.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_heap.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_iterator.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_iterator_base_funcs.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_iterator_base_types.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_list.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_map.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_multimap.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_multiset.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_pair.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_relops.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_set.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_tempbuf.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stl_tree.h \\
/include/c++/`cat gcc/BASE-VER`/bits/stringfwd.h \\
/include/c++/`cat gcc/BASE-VER`/debug/debug.h \\
/include/c++/`cat gcc/BASE-VER`/ext/alloc_traits.h \\
/include/c++/`cat gcc/BASE-VER`/ext/atomicity.h \\
/include/c++/`cat gcc/BASE-VER`/ext/new_allocator.h \\
/include/c++/`cat gcc/BASE-VER`/ext/numeric_traits.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/atomic_word.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/c++allocator.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/c++config.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/c++locale.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/cpu_defines.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/gthr-default.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/gthr.h \\
/include/c++/`cat gcc/BASE-VER`/`./config.guess`/bits/os_defines.h \\
/lib/cpp \\
/lib/libgcc_s.so \\
/lib/libstdc++.so \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtbegin.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtbeginS.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtend.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtendS.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/crtfastmath.o \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/libgcc.a \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/cpuid.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/emmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/float.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/ia32intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/immintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/mm_malloc.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/mmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/smmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdarg.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdbool.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stddef.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/stdnoreturn.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/tmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/unwind.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/x86intrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include/xmmintrin.h \\
/lib/gcc/`./config.guess`/`cat gcc/BASE-VER`/include-fixed/limits.h \\
/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`/cc1 \\
/libexec/gcc/`./config.guess`/`cat gcc/BASE-VER`/cc1plus

exec=\\
/lib/libgcc_s.so.1 \\
/lib/libstdc++.so.6
EOF

exit
--- gcc-6.5.0.orig/gcc/config/i386/t-linux64
+++ gcc-6.5.0/gcc/config/i386/t-linux64
@@ -33,6 +33,6 @@
 comma=,
 MULTILIB_OPTIONS    = $(subst $(comma),/,$(TM_MULTILIB_CONFIG))
 MULTILIB_DIRNAMES   = $(patsubst m%, %, $(subst /, ,$(MULTILIB_OPTIONS)))
-MULTILIB_OSDIRNAMES = m64=../lib64$(call if_multiarch,:x86_64-linux-gnu)
+MULTILIB_OSDIRNAMES = m64=../lib$(call if_multiarch,:x86_64-linux-gnu)
 MULTILIB_OSDIRNAMES+= m32=$(if $(wildcard $(shell echo $(SYSTEM_HEADER_DIR))/../../usr/lib32),../lib32,../lib)$(call if_multiarch,:i386-linux-gnu)
 MULTILIB_OSDIRNAMES+= mx32=../libx32$(call if_multiarch,:x86_64-linux-gnux32)
